CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
-## v0.4.7
+## v0.4.8
```

Comment:
Is it worth documenting somewhere in the changelog that 0.4.7 was a bad release? I know that this is intended to be a release PR, but figured it was worth raising this topic in the event that it hasn't been already @izakp

Reply:
Ah yeah good idea

Reply:
https://github.com/artsy/hokusai/commit/13ad18c52d01b24261014b45f6d53e06841ca239